networkit/cpp/generators/test/GeneratorsGTest.cpp

```diff
@@ -914,25 +934,24 @@ TEST_F(GeneratorsGTest, tryGiganticCollectionOfHyperbolicUnitDiskGraphs) {
 }
 
 TEST_F(GeneratorsGTest, testLFRGenerator) {
-	Aux::Random::setSeed(42, true);
```

Comment:
Not using a static seed for the LFR generator test case is a very bad idea as the parameters are relatively tight and depending on the actual randomness used this test case might fail randomly because the drawn sequence is not realizable. I understand that this might not be desired that an algorithm might fail randomly but imho it is better to fail instead of trying to repair the sequence as it is not easy to see if a failure is (almost) unavoidable with the given parameters or if failure is unlikely. The original LFR generator does some strange repairing operations that might give you a completely different community size sequence than what you expected.

Reply:
I see your point, in this specific case we might need a solution like explained in https://github.com/kit-parco/networkit/pull/170#discussion_r195463533. 

The question is, is the algorithm really implemented to fail randomly? If so, then the seed should definitely set here back and this behaviour should be written down in the comments. This would not be a desirable behaviour if used in a productive environment and has to be caught there.

Reply:
I have opened #171 to discuss this in more detail but would suggest to not to remove this seed in this PR.

Reply:
https://github.com/kit-parco/networkit/pull/170/commits/75c0e0fd4cc6761505f8e0027ee8be0ed1469315 reverts the changes